### PR TITLE
Event View Model Details schema for validation and documentation

### PR DESCRIPTION
**Pull request by remi00**
### What it does?

- Event View Model Details schema with possible detail types, describing all their important information
- Use that schema for creating documentation Markdown for users reference
- Test it to make sure future updates don't break it

### What else is worth noting?

- This is missing piece to EventViewModel approach for handling notification request that addresses all recent challenges:
  - The flexibility of the EVM which is helpful in combining the data model for various notification sources made it easy that format of notifications could break when some detail type or value changed.
  - There was missing full documentation of all possible details that users could use for notifications.
- Next step: define all other missing EventViewModel details, mostly for synthetics
- See also Notify PR: https://github.com/kentik/notify/pull/341

### Overview

image

### CODE REVIEW

**Review by durakup — DISMISSED** [review]
Lgtm!

**Review by zamkot — DISMISSED** [review]

File: schemas/details.yaml
Comment (elodszopos) [inline schemas/details.yaml:1]: This is awesome!

**Review by elodszopos — DISMISSED** [review]
Beautiful!

**Comment by rchapin** [comment]
Are the "Example values" and "Value schema" columns mixed up?
image

**Comment by remi00** [comment]
> Are the "Example values" and "Value schema" columns mixed up?

You are right they were mixed up 👍

**Review by elodszopos — APPROVED** [review]

**Pull request merged**